core/server/translations/en.json

```diff
@@ -171,7 +171,8 @@
         },
         "general": {
             "moreInfo": "\nMore info: {info}",
-            "requiredOnFuture": "This will be required in future. Please see {link}"
+            "requiredOnFuture": "This will be required in future. Please see {link}",
+            "database": "Database problem, please try again or contact our support."
```

Comment:
Whatever we do with this fix, if we keep an error message here it needs to be rewritten to be clearer what the problem is and why it happened and what to do to fix it (contacting support is not going to help if there is no explanation).